### ISSUE TITLE
Access to ...googlevideo.com was denied with AWS Lambda

### ISSUE THREAD

**Issue by luis2villalba** [yt-dlp/yt-dlp#9015]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm reporting that yt-dlp is broken on a **supported** site
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've checked that all URLs and arguments with special characters are [properly quoted or escaped](https://github.com/yt-dlp/yt-dlp/wiki/FAQ#video-url-contains-an-ampersand--and-im-getting-some-strange-output-1-2839-or-v-is-not-recognized-as-an-internal-or-external-command)
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

US East North Virginia

### Provide a description that is worded well enough to be understood

Hi every one, I hope you can help me with this issue. 

First of all to give a general context, my idea with this library or the reason becasuse I'm using for is for getting an audio link from a youtube url video. 

The problem is when I try to get the url from a Lambda which it's deployed in an AWS server, for some URLs I'm getting this error, it only happen with some urls no with all of them. 

The error is: 
**Access to rr1---sn-p5qlsn76.googlevideo.com was denied
You don't have authorization to view this page.
HTTP ERROR 403**

However when I try to get the same URL locally or from my docker enviroment locally using this image: public.ecr.aws/lambda/python:3.12 it works perfectly in all cases. 

For instance the url that I'm getting for this url: http://www.youtube.com/watch?v=Zc13NAc7dvM
Is this: 
https://rr2---sn-p5qlsnrl.googlevideo.com/videoplayback?expire=1705460862&ei=HvCmZYz3FIqI_9EPwsu5wA8&ip=34.204.206.220&id=o-AOCT8ls7Wd0Lb7Y2Coqi_O_RYWzpw66VxCD_4aEYEYPA&itag=139&source=youtube&requiressl=yes&xpc=EgVo2aDSNQ%3D%3D&mh=ZZ&mm=31%2C29&mn=sn-p5qlsnrl%2Csn-p5qs7nzy&ms=au%2Crdu&mv=m&mvi=2&pl=18&gcr=us&initcwndbps=1300000&vprv=1&svpuc=1&mime=audio%2Fmp4&gir=yes&clen=1633515&dur=267.679&lmt=1699185911354643&mt=1705439089&fvip=3&keepalive=yes&fexp=24007246&c=IOS&txp=5532434&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cxpc%2Cgcr%2Cvprv%2Csvpuc%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=AJfQdSswRQIhAIcuKeXnf3m7flD_hpvKGgQEHl_9Z-agROIKLsNE11DhAiBAZkGLNVnsb3MIl3ztauo_poI3GzdCvaoM8W_7qJZZbA%3D%3D&lsparams=mh%2Cmm%2Cmn%2Cms%2Cmv%2Cmvi%2Cpl%2Cinitcwndbps&lsig=AAO5W4owRgIhAIPMq_TXW7TejTUnPUYHFaoKHRMrS43eUG7bomo3iZS3AiEAxLxz5OvdBO4SUxpn7BiGQsJ-5VvH6mQlYe0cqKLzUQE%3D

If I try to acces I get a denied error. 

It's an strage issue because only hapen with some videos others links works perfectly in both enviroments (local and AWS Lambda). 

Thanks in advance for any suggestions that you can give me to solve this

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[INFO]	2024-01-16T21:07:36.467Z	e435711d-fa27-47cf-b8be-765db48e9303	Looking for: Gilberto Santa Rosa
[INFO]	2024-01-16T21:07:39.572Z	e435711d-fa27-47cf-b8be-765db48e9303	Getting youtube-dl url for https://www.youtube.com/watch?v=Zc13NAc7dvM
[debug] Encodings: locale utf-8, fs utf-8, pref utf-8, out utf-8 (No ANSI), error utf-8 (No ANSI), screen utf-8 (No ANSI)
[debug] yt-dlp version nightly@2024.01.14.232710 from yt-dlp/yt-dlp-nightly-builds [8e6e36517] (pip) API
[debug] params: {'quiet': True, 'verbose': True, 'ignoreerrors': False, 'cachedir': None, 'outtmpl': '%(title)s.%(ext)s', 'format': 'm4a/bestaudio/best', 'postprocessors': [{'key': 'FFmpegExtractAudio', 'preferredcodec': 'm4a'}], 'compat_opts': set(), 'http_headers': {'User-Agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/97.0.4692.20 Safari/537.36', 'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8', 'Accept-Language': 'en-us,en;q=0.5', 'Sec-Fetch-Mode': 'navigate'}}
[debug] Python 3.12.0 (CPython x86_64 64bit) - Linux-5.10.199-210.747.amzn2.x86_64-x86_64-with-glibc2.34 (OpenSSL 3.0.8 7 Feb 2023, glibc 2.34)
[debug] exe versions: none
[debug] Optional libraries: Cryptodome-3.20.0, certifi-2023.11.17, mutagen-1.47.0, requests-2.31.0, sqlite3-3.40.0, urllib3-2.0.7, websockets-12.0
[debug] Proxy map: 
{}

[debug] Request Handlers: urllib, requests, websockets
[debug] Loaded 1798 extractors
[youtube] Extracting URL: http://www.youtube.com/watch?v=Zc13NAc7dvM
[youtube] Zc13NAc7dvM: Downloading webpage
[youtube] Zc13NAc7dvM: Downloading ios player API JSON
[youtube] eWHYoHISSNs: Downloading android player API JSON
[youtube] eWHYoHISSNs: Downloading m3u8 information
[debug] Sort order given by extractor: quality, res, fps, hdr:12, source, vcodec:vp9.2, channels, acodec, lang, proto
[debug] Formats sorted by: hasvid, ie_pref, quality, res, fps, hdr:12(7), source, vcodec:vp9.2(10), channels, acodec, lang, proto, size, br, asr, vext, aext, hasaud, id
```

**Comment by bashonly** [comment]
You're getting the 403 because the AWS lambda instance's IP address is blocked by youtube

Duplicate of #7860
